modules/accredited_representative_portal/app/controllers/accredited_representative_portal/v0/power_of_attorney_request_decisions_controller.rb

```diff
@@ -18,13 +18,14 @@ class PowerOfAttorneyRequestDecisionsController < ApplicationController
 
       def create
         reason = decision_params[:reason]
+        declination_reason = decision_params[:declination_reason]
```

Comment:
We should take over the `reason` parameter!

Reply:
What does that mean?